smoketest/alertlog_test.go

```diff
@@ -0,0 +1,227 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/smoketest/harness"
+)
+
+func TestAlertLog(t *testing.T) {
+	t.Parallel()
+
+	type alertLogs struct {
+		Alert struct {
+			RecentEvents struct {
+				Nodes []struct {
+					Message string
+					State   struct {
+						Details string
+					}
+				}
+			}
+		}
+	}
+
+	type config struct {
+		CMDisabled bool
+		CMType     string
+		NR         bool
+		EPStep     bool
+		EPStepUser bool
+	}
+
+	doQL := func(h *harness.Harness, query string, res interface{}) {
```

Comment:
This will need `t *Testing.T` as an argument so that it uses the `t` from `t.Run` and not the top-level test instance.

```suggestion
	doQL := func(t *testing.T, h *harness.Harness, query string, res interface{}) {
```